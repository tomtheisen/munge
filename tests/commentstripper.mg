(
    ! various string literals
    /(["'`])(?:\\.|.)*?\1/s => ()

    ! comments
    /\/\/.*/ => ""
    /\/\*.*?\*\//s => ""
)