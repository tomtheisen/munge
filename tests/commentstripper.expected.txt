console.log("foo"); 



console.log("baz"); 
const real = code;


const weird = " \" // "; 
const multiline = `
// not actually a comment
content
/*
not actually a block comment
*/
` +  "suffix";
